dbz-engine/src/main/java/com/example/DebeziumRunner.java

```diff
@@ -400,6 +401,7 @@ public void markBatchComplete(int batchid, boolean markall, int markfrom, int ma
 			myBatch.committer.markBatchFinished();
 			activeBatchHash.remove(batchid);
 		}
+		System.gc();
```

Comment:
Consider adding comments explaining why System.gc() is needed

Reply:
System.gc() basically forces Java to do a garbage collect to reclaim heap memory space immediately.